main.py

```diff
@@ -340,6 +340,9 @@ def ui_access_wrapper(self, *args, **kwargs):
     get_redirect_route(
         r'%s/story/<story_url_fragment>' % feconf.TOPIC_VIEWER_URL_PREFIX,
         story_viewer.StoryPage),
+    get_redirect_route(
```

Comment:
This seems to be placed somewhat randomly in the list. Put this near the contributor dashboard URLs, and follow the existing convention of having the names defined as constants in feconf. Follow existing naming patterns/styles.

Reply:
Fixed, but I'm a little confused what the convention is, some urls seem to use feconf constants while others do not.

Reply:
Yeah, that's fair -- I forgot we hadn't fixed this universally yet. What you have for now is fine for now, I think -- thanks!